tracing-core/src/field.rs

```diff
@@ -219,6 +219,8 @@ pub trait Value: crate::sealed::Sealed {
 }
 
 /// A `Value` which serializes as a string using `fmt::Display`.
+/// Uses `record_debug` in the `Value` implementation to
```

Comment:
Nit: can we put a newline between this and the first line of the doc comment? I try to follow https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#summary-sentence whenever possible...